### PR TITLE
Feature/web 987 pdp products carousel

### PR DESCRIPTION
**Pull request by Hanumant-Bangar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
     "npm": ">=8.3.0"
   },
   "scripts": {
-    "dev": "NODE_OPTIONS='-r next-logger' next dev",
+    "dev": "next dev",
```
Comment (poonamg-ign) [inline package.json]: Please revert this line

File: package.json [hunk 2]
```diff
@@ -5,9 +5,9 @@
...
     "build": "next build",
-    "start": "NODE_OPTIONS='-r next-logger' next start",
+    "start": "next start",
```
Comment (poonamg-ign) [inline package.json]: Please revert this line

File: src/pages/product/[productCode].tsx [hunk 3]
```diff
@@ -67,6 +68,32 @@ export async function getStaticProps(
   const { locale, params } = context
   const { productCode } = params as any
   const product = await getProduct(productCode)
+  const relatedProducts = []
+  const relatedProductData = product?.properties?.find(
+    (item: any) => item.attributeFQN === 'tenant~related-products'
+  )?.values[0].stringValue
```
Comment (poonamg-ign) [inline src/pages/product/[productCode].tsx]: what if values is null? add proper conditions

File: package-lock.json
Comment (poonamg-ign) [inline package-lock.json:1]: do not commit this file, only package.json suffice.

**Review by poonamg-ign — CHANGES_REQUESTED** [review]
Do not commit package-lock.json file

File: package.json (hunk 1 above)
Comment (Hanumant-Bangar) [reply package.json]: changes done

File: src/pages/product/[productCode].tsx (hunk 3 above)
Comment (Hanumant-Bangar) [reply src/pages/product/[productCode].tsx]: changes done

**Pull request merged**